modules/rubiconBidAdapter.js

```diff
@@ -542,42 +543,47 @@ export const spec = {
     if (bidRequest?.ortb2Imp?.ext?.ae) {
       data['o_ae'] = 1;
     }
+    // If the bid request contains a 'mobile' property under 'ortb2.site', add it to 'data' as 'p_site.mobile'.
+    if (bidRequest?.ortb2?.site?.mobile) {
```

Comment:
If defined `mobile` is either `0` or `1`

So, with this current code it would only pass it if it was `1` since `0` is falsy.

So lets add typeof here and pass along.

```js
if (typeof bidRequest?.ortb2?.site?.mobile === 'number') {
```

Also add a test to confirm `p_site.mobile`